notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1061 @@
+{
```

Comment:
<p>Is this a configuration only for electric field reader? if YES, please move it to section 3. I think we can only config on some common parameters here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>